code/index.html

```diff
@@ -4,34 +4,67 @@
     <link rel="stylesheet" href="style.css" />
-    <title>Technigo Project Cluedo</title>
+    <title>Murder Mystery</title>
   </head>
   <body>
+    <header>
+      <h1>Create your own cluedo mystery!<img class="magnifying-icon" src="assets/icons8-magnifying-glass-tilted-left-48.png"></h1>
+      <p>Click the cards to generate a killer, weapon and room</p>
+    </header>
+
+
     <section class="cards">
       <article class="killer-deck" id="killerCard">
-        <p>KILLER</p>
-        <p id="killerName"></p>
-        <p id="killerAge"></p>
-        <p id="killerOccupation"></p>
-        <img id="killerImage" src="" height="100" />
-        <p id="killerDescription"></p>
+        <div class="killer-info-wrapper" id="killer-info-wrapper">
+          <p class="start-view-cards" id="start-view-card-killer">PICK KILLER</p>
+          <p id="killerName"></p>
+          <p id="killerAge"></p>
+          <p id="killerOccupation"></p>
+          <img id="killerImage" src="" height="100" />
+          <p id="favoriteWeapon"></p>
+          <p id="killerDescription"></p>
+        </div>
+        <div class="loader-wrapper" id="loader-wrapper-killer">
+          <div class="loader"></div>
+          <p class="generating-text">Generating a suspect</p>
+        </div>
       </article>
       <article class="weapon-deck" id="weaponCard">
-        <p>WEAPON</p>
-        <p id="weaponName"></p>
-        <p id="weaponWeight"></p>
+        <p class="start-view-cards" id="start-view-card-weapon">PICK WEAPON</p>
+        <div class="weapon-info-wrapper" id="weapon-info-wrapper">
+          <p id="weaponName"></p>
+          <p id="weaponWeight"></p>
+        </div>
+        <div class="loader-wrapper" id="loader-wrapper-weapon">
+          <div class="loader"></div>
+          <p>Generating a weapon</p>
+        </div>
       </article>
       <article class="room-deck" id="roomCard">
-        <p>ROOM</p>
-        <p id="roomName"></p>
+        <p class="start-view-cards" id="start-view-card-room">PICK ROOM</p>
+        <div class="room-info-wrapper" id="room-info-wrapper">
+          <p id="roomName"></p>
+        </div> 
+        <div class="loader-wrapper" id="loader-wrapper-room">
+          <div class="loader"></div>
+          <p>Generating a room</p>
+        </div>
       </article>
     </section>
 
-    <h1 id="mystery"></h1>
+    <section class="revelation">
+      <h2 id="mystery"></h2>
+      <button id="revealButton">REVEAL THE CRIME</button>
+    </section>
 
-    <button>REVEAL THE CRIME</button>
+    <footer>
+      <p>Project 4, Cluedo Mystery Picker, by Anna Hellqvist @ Technigo Bootcamp 2020</p>
+      <p><a target="_blank" href="https://icons8.com/icons/set/magnifying-glass-tilted-left-emoji">Magnifying Glass Tilted Left icon</a> icon by <a target="_blank" href="https://icons8.com">Icons8</a></p>
+    </footer>
 
+  
     <script src="clue.js"></script>
   </body>
 </html>
```

Comment:
Nicely structured and clean code!✌ Just think about adding ./ first in the file path, like "./clue.js

Reply:
done! :)